modules/nextflow/src/main/groovy/nextflow/config/ConfigBuilder.groovy

```diff
@@ -787,7 +787,7 @@ class ConfigBuilder {
                 return true
 
             def result = process
-                            .findAll { String name, value -> name.startsWith('$') && value instanceof Map }
+                            .findAll { String name, value -> name.startsWith('withName:') && value instanceof Map }
```

Comment:
Does this not break `$`-prefixed config?

Reply:
It will cause containerized execution to fail unless a global container is specified. That is currently what happens with the `withName:` syntax. We can keep both if you want

Reply:
Done